website/docs/src/content/component-reference/storage-components/database-reference.mdx

```diff
@@ -0,0 +1,550 @@
+---
+title: SQL database reference
+description: SQL database components for persistent data in GenSX workflows.
+---
+
+# Database reference
+
+API reference for GenSX Cloud SQL database components.
+
+## Installation
+
+```bash
+npm install @gensx/storage
+```
+
+## DatabaseProvider
+
+Provides SQL database capabilities to its child components.
+
+### Import
+
+```tsx
+import { DatabaseProvider } from "@gensx/storage";
+```
+
+### Props
+
+| Prop            | Type                  | Default     | Description                                      |
+| --------------- | --------------------- | ----------- | ------------------------------------------------ |
+| `kind`          | `"filesystem" \| "cloud"` | `"cloud"` | The storage backend to use                   |
```

Comment:
the default here isn't actually correct. We should say somewhere in here that if you don't specify, we use local for dev and cloud for prd.